src/applications/ivc-champva/10-10D/helpers/useBrowserMonitoring.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { datadogRum } from '@datadog/browser-rum';
+
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+import { makeSelectFeatureToggles } from '../selectors/feature-toggles';
+
+const initializeRealUserMonitoring = () => {
+  // Prevent RUM from re-initializing the SDK OR running on local/CI environments.
+  if (
+    !environment.BASE_URL.includes('localhost') &&
+    !window.DD_RUM?.getInitConfiguration()
+  ) {
+    datadogRum.init({
+      applicationId: 'cca24a05-9ea0-49ea-aaa9-0d1e04a17ba0',
+      clientToken: 'puba5e0866f8008f60a6bc8b09ae555dd92',
+      site: 'ddog-gov.com',
+      service: '10-10d',
+      env: environment.vspEnvironment(),
+      sessionSampleRate: 100,
+      sessionReplaySampleRate: 100,
```

Comment:
Defaulting to 100% because we need to test in staging. Once we decide to start using in prod, this will have to change to a significantly reduced percentage.